scripts/acrobot.coffee

```diff
@@ -0,0 +1,33 @@
+# Description:
+#   checks an acronym against a set of acronyms used at CFPB and provides its meaning
+#
+# Dependencies:
+#   local json file "acro.json"
+#
+# Commands:
+#   hubot define <term> - returns an acronym's meaning if it's in acro.json
+#
+# Author:
+#   higginsw
+
+module.exports = (robot) ->
+  
+  fs = require 'fs'
+  acroPath = process.cwd()+'/acro.json'
+  acroList = JSON.parse(fs.readFileSync(acroPath, 'utf8'))
+
+  robot.respond /define (.*)$/i, (res) ->
+    term = res.match[1].toUpperCase()
+    if term of acroList
+      res.send buildAnswer(term) 
+    else
+      res.send "Sorry, can't find #{term}"
+
+  buildAnswer = (term) ->
+    acroObj = acroList[term]
+    answer = "#{acroObj.acro} stands for #{acroObj.name}"
+    if acroObj.note
+      answer = answer + " — " + acroObj.note
+    if acroObj.link
+      answer = answer + " #{acroObj.link}"
```

Comment:
It would've been nice to have the link in a tag, but alas. I think the dash would be nice, because without it the link will now just run right into the back of the note (or name, if no note is present). True, there is one space, but I think a more explicit separator would be nice. Just my opinion. Wouldn't hold up the PR for it.